extensions/jackson/spi/src/main/java/io/quarkus/jackson/spi/CustomJacksonSerDeserBuildItem.java

```diff
@@ -0,0 +1,67 @@
+package io.quarkus.jackson.spi;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * BuildItem that will be used to customize the ObjectMapper (produced by io.quarkus.jackson.ObjectMapperProducer)
+ * with custom serializers and/or deserializers
+ *
+ * Serializers and deserializers MUST contain a public no-args constructor
+ */
+public final class CustomJacksonSerDeserBuildItem extends MultiBuildItem {
```

Comment:
So reading that, I wonder if it should be `JacksonModuleBuildItem` instead and have proper methods to add serializer/deserializers to the module.

Reply:
That sounds like an excellent idea!

Reply:
Actually, do we have any examples of making the BuildItem mutable? I don't recall seeing that being done before.